### PR TITLE
Update v6.3.5

### PR DESCRIPTION
**Pull request by pabloverduzco**
The main motivation for this change was that the link for the GitHub Light High Contrast theme was redirecting to a 404 (Not Found) error. Upon investigation, it was discovered that this was due to a typo in the filename.

## Changes
- Renamed 'GitHub Light High Constrast.itermcolors' to 'GitHub Light High Contrast.itermcolors' to fix the typo.
- Updated all links in README.md to point to the correct repository ('github-vscode-theme-iterm' instead of 'github-theme-iterm').

### CODE REVIEW

**Comment by cdalvaro** [comment]
Hi @pabloverduzco! Thank you for opening this issue!

I'm not sure why you have closed it, since you were right renaming the file!

**Comment by pabloverduzco** [comment]
Hello @cdalvaro! I hope you're doing well. I apologize for closing the PR. I was actually about to open a new one with signed commits since this one is blocked for that reason. Do you have any suggestions on how I should proceed? I'd appreciate your input on this.

**Comment by cdalvaro** [comment]
Hi @pabloverduzco, feel free to make a `git push --force` to this branch or create a new PR. Whatever you prefer!

**Comment by pabloverduzco** [comment]
Hello again, @cdalvaro. I apologize for the inconvenience with the commit signatures. I was experiencing some issues with my GPG key configuration, which is why I force-pushed the commits multiple times. Thank you for your patience. I'm excited to contribute to this awesome project!

**Comment by cdalvaro** [comment]
@pabloverduzco, could you change all references to version `6.3.4` in the [README.md](https://github.com/cdalvaro/github-vscode-theme-iterm/blob/main/README.md) file to point to the new one?

**Comment by pabloverduzco** [comment]
Thank you for pointing that out @cdalvaro. I've already updated all references to version 6.3.4 in the README.md file to the new version in my latest commit.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # GitHub VS Code Theme for iTerm
 
-<img src="https://github.gallerycdn.vsassets.io/extensions/github/github-vscode-theme/6.3.4/1680003819182/Microsoft.VisualStudio.Services.Icons.Default" width="64px" align="right" style="top: 1px; visibility: visible;" />
+<img src="https://github.gallerycdn.vsassets.io/extensions/github/github-vscode-theme/6.3.5/1680003819182/Microsoft.VisualStudio.Services.Icons.Default" width="64px" align="right" style="top: 1px; visibility: visible;" />
```
Comment (cdalvaro) [inline README.md]: It seems that this image hasn't been updated in the market place, so this change must be undone.

The rest of the changes are OK!

Comment (pabloverduzco) [reply README.md]: You're absolutely right. I didn't notice that this was an external asset reference. Sorry about that.

**Comment by cdalvaro** [comment]
Thank you for your contribution @pabloverduzco!

**Pull request merged**